Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseTeleOp.java

```diff
@@ -97,7 +97,7 @@ public void driveGrabberWithController() {
      */
     public void driveSlidesWithController() {
         // joystick control
-        slideTargetPosition += (int) (-gamepad2.left_stick_y * 25);
+        slideTargetPosition += (int) (-gamepad2.left_stick_y * 10);
```

Comment:
10 is a magic number here. What is the multiplier for?

Reply:
Replaced with constant